js/avatarMovement.js

```diff
@@ -1,76 +1,49 @@
-//player A variables
-var playerAx = 150;
-var playerAy = 150;
-var playerAHeight = 50;
-var playerAWidth = 50;
-var playerAOrientation = 'E';
-
-//Player B variables
-var playerBx = 550;
-var playerBy = 550;
-var playerBHeight = 50;
-var playerBWidth = 50;
-var playerBOrientation = 'W';
-
-//create players
-var playerA = drawPlayer(playerAx, playerAy, playerAHeight, playerAWidth, 'pink', 'playerA', playerAOrientation);
-var playerB = drawPlayer(playerBx, playerBy, playerBHeight, playerBWidth, 'blue', 'playerB', playerBOrientation);
+var countdownStart = 3;
+var startGame = false;
 
 //gameplay variables
 var distance = 1000;
 var time = 3000;
 
-function drawPlayer(x, y, h, w, fill, id, orientation) {
-    var rect = document.createElementNS("http://www.w3.org/2000/svg", "rect");
-    rect.setAttributeNS(null, 'x', x);
-    rect.setAttributeNS(null, 'y', y);
-    rect.setAttributeNS(null, 'height', h);
-    rect.setAttributeNS(null, 'width', w);
-    rect.setAttributeNS(null, 'fill', fill);
-    rect.setAttributeNS(null, 'id', id);
-    rect.setAttributeNS(null, 'orientation', orientation);
-    document.getElementById('svgOne').appendChild(rect);
-    return document.getElementById(id);
-}
+var $playerA = $('#playerA');
```

Comment:
do you want the player variables to start with a $?

Reply:
this is here to clarify that we using JQuery for these variables